windows/RNFS.Tests/ReactNativeCxxTests/JsonJSValueReader.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "JsonReader.h"
+#include "winrt/Microsoft.ReactNative.h"
+
+namespace winrt::Microsoft::ReactNative {
+
+struct JsonJSValueReader : implements<JsonJSValueReader, IJSValueReader> {
```

Comment:
This seems like it's re-implementing JSON parsing... we should just use the Windows.Data.Json WinRT APIs directly instead

Reply:
@avmoroz just making sure you saw this comment, do we need this custom json parser?

Reply:
To be completely honest, I asked @vmoroz about a way to create unit tests for my new implementations of RNFS functions, and he recommended I test the functions like this. This custom json parser is solely there for the unit tests.